internal/parser/parser.go

```diff
@@ -0,0 +1,47 @@
+package parser
```

Comment:
Nit: I like "parse" for the package. There's a tendency towards using verb-er for everything.